docs/configuration.asciidoc

```diff
@@ -243,7 +244,7 @@ The default unit for this option is `s`.
 If set to `true`,
 the agent will capture request and response headers, including cookies.
 
-NOTE: Setting this to `false` reduces network bandwidth, disk space and object allocations.
+NOTE: Setting this to `false` reduces memory allocations, network bandwidth and Elasticsearch disk space.
```

Comment:
Would maybe `Elasticsearch disk space` -> `disk space used by Elasticsearch` better?

Reply:
I agree - `disk space used by Elasticsearch` sounds better. I've changed it.